scripts/migrate-redis-keys.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
```

Comment:
@alexluong - we wanted to delete this, right, and use the Outpost CLI?

Reply:
yes, I would say so